helpers/website_helper.py

```diff
@@ -18,3 +21,17 @@ def format_url(cls, website_url):
             formatted_url = website_url
 
         return formatted_url
+
+    @classmethod
+    def exists(cls, website_url):
+        """
+        Verify that a given URL exists (returns a non-404 status code)
+        """
+        context = ndb.get_context()
+
+        if not website_url:
+            return False
+
+        result = context.urlfetch(website_url).get_result()
+
+        return int(result.status_code) != 404
```

Comment:
Is the status code not already an int?

Reply:
Good point, I'm guessing you're right. For some reason I thought it was a string, but I'll check.